backend/src/main/scala/cromwell/backend/standard/callcaching/StandardFileHashingActor.scala

```diff
@@ -80,10 +87,36 @@ abstract class StandardFileHashingActor(standardParams: StandardFileHashingActor
   override lazy val serviceRegistryActor: ActorRef = standardParams.serviceRegistryActor
   override lazy val configurationDescriptor: BackendConfigurationDescriptor = standardParams.configurationDescriptor
 
+  // Child classes can override to set per-filesystem defaults
+  val defaultHashingStrategies: Map[String, FileHashStrategy] = Map.empty
+
+  // Hashing strategy to use if none is specified.
+  val fallbackHashingStrategy: FileHashStrategy = FileHashStrategy.Md5
+
+  // Combines defaultHashingStrategies with user-provided configuration
+  lazy val hashingStrategies: Map[String, FileHashStrategy] = {
+    val configuredHashingStrategies = for {
+      fsConfigs <- configurationDescriptor.backendConfig.as[Option[Config]]("filesystems").toList
+      fsKey <- fsConfigs.root.keySet().asScala
+      configKey = s"${fsKey}.caching.hashing-strategy"
+      // TODO this allows users to override with an empty list to prevent caching, is that desirable or a footgun?
```

Comment:
There is already a global `call-caching.enabled` flag that seems like a better choice.

I don't it makes sense to let users toggle call caching on a per-filesystem basis because one backend can use multiple filesystems in the same task (e.g. GCS + DRS).

I would make empty list an exception if `call-caching.enabled` is `true`.

Reply:
Good point, will change.